src/actions/sources.js

```diff
@@ -174,19 +174,28 @@ export function selectSourceURL(
   };
 }
 
+declare type LocationObject = {
+  id?: ?string,
+  sourceId?: ?string,
+  tabIndex?: ?number,
+  line?: ?number,
+  column?: ?number,
+  sourceUrl?: ?string
+};
+
 /**
  * @memberof actions/sources
  * @static
  */
-export function selectSource(id: string, options: SelectSourceOptions = {}) {
+export function selectLocation(location: LocationObject) {
```

Comment:
@lukaszsobek for now don't replace this function as it is used in MC here 
https://dxr.mozilla.org/mozilla-central/source/devtools/client/shared/view-source.js#65
maybe just create a new `selectLocation` function and call `selectSource` from there

Reply:
Might have to change that eventually

Reply:
@lukaszsobek sorry! ignore this comment here... i've move the comment up to the right location [here](https://github.com/devtools-html/debugger.html/pull/4789/files#r153234382)